src/DearImGui/Raw.hs

```diff
@@ -245,6 +253,48 @@ C.context (Cpp.cppCtx <> C.bsCtx <> imguiContext)
 C.include "imgui.h"
 Cpp.using "namespace ImGui"
 
+-- | Wraps @ImGuiListClipper
+newtype ImGuiListClipper = ImGuiListClipper (Ptr ())
+
+
+-- | Wraps @ImGuiListClipper::Begin()
+-- 
+-- The pointer has to be deleted manually by destroiyImGuiListClipper
+beginListClipper :: MonadIO m => Int -> m ImGuiListClipper
+beginListClipper size 
+  | size < 0 = error "Size of the Clipper can't be negative"
```

Comment:
Size would be better enforced by type system in non-raw wrapper by using `Natural` type.
Raw code should stick to C types, namely CInt and just pass it to the code.